pkg/client/generated/management/v3/zz_generated_node_group.go

```diff
@@ -33,6 +34,7 @@ type NodeGroup struct {
 	LaunchTemplate       *LaunchTemplate   `json:"launchTemplate,omitempty" yaml:"launchTemplate,omitempty"`
 	MaxSize              *int64            `json:"maxSize,omitempty" yaml:"maxSize,omitempty"`
 	MinSize              *int64            `json:"minSize,omitempty" yaml:"minSize,omitempty"`
+	NodeRole             *string           `json:"nodeRole,omitempty" yaml:"nodeRole,omitempty"`
```

Comment:
Did we miss a go generate or something? Where is this coming from?

Reply:
It would appear that way, https://github.com/rancher/eks-operator/pull/73 introduced node role but a corresponding go generate was never run after merging into rancher: https://github.com/rancher/rancher/pull/39881